openshift/templates/clamav/lup-clamav-deployment.yaml

```diff
@@ -0,0 +1,97 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: lup-clamav
+  namespace: e8b9ad-tools
+  annotations:
+    deployment.kubernetes.io/revision: '7'
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: lup-clamav
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: lup-clamav
+    spec:
+      volumes:
+        - name: tmp
+          emptyDir: {}
+        - name: clamav-lib
+          emptyDir: {}
+        - name: clamav-log
+          emptyDir: {}
+        - name: runclamav
+          emptyDir: {}
+      containers:
+        - resources: {}
+          terminationMessagePath: /dev/termination-log
+          name: clamd
+          command:
+            - /bin/sh
+          env:
+            - name: CLAMD_CONFIG_FILE
+              value: /etc/clamav/clamd.conf
+          securityContext:
+            capabilities:
+              drop:
+                - ALL
+            allowPrivilegeEscalation: false
+          ports:
+            - containerPort: 3310
+              protocol: TCP
+          imagePullPolicy: IfNotPresent
+          volumeMounts:
+            - name: clamav-lib
+              mountPath: /var/lib/clamav
+            - name: clamav-log
+              mountPath: /var/log/clamav
+            - name: tmp
+              mountPath: /tmp
+            - name: runclamav
+              mountPath: /run/clamav
+          terminationMessagePolicy: File
+          image: 'clamav/clamav:latest'
```

Comment:
We should pick a version for the image to decrease the chance of an updated image breaking something the next time we do a rollout

Reply:
Sure, I can do that. My only concern is that the software could eventually become too outdated to get fresh virus definition updates, which would obviously be bad. However, we're supposed to be sunsetting this platform soon, so I guess we're thinking more for the short term.

Reply:
Glad to hear your reasoning behind the choice and I can understand that. In my opinion, the approach to dependency management is always better met with certainty over fluidity. When we pick an exact version, we can usually expect at least a year of support before we have to upgrade and we know exactly what to expect from the library in terms of its behaviour.

If we don't know which version we're using, then we can't always depend on exactly the same behaviour. So on the flip side, maybe a new version released tomorrow changes how the library queries the `clamd` daemon and that ends up breaking scanning. When you pick a discrete version, you know you can bet on it for an interval of time